src/Ledger/GovernanceActions.lagda

```diff
@@ -185,11 +200,13 @@ submitted. Beside the proposed action, it requires:
 \item an \Anchor, providing further information about the proposal.
 \end{itemize}
 
-The deposit will be returned when the action is removed from the state
-in any way. It will be added to the deposit pot, similar to stake key
-deposits. It will also be counted towards the stake of the reward
-address it will be paid back to, to not reduce the submitter's voting
-power to vote on their own (and competing) actions.
+While the deposit is being held, is added to the deposit pot similar
+to stake key deposits. It is also counted towards the stake of the
+reward address it will be paid back to, to not reduce the submitter's
+voting power to vote on their own (and competing) actions. For a
```

Comment:
```suggestion
reward address to which it will be returned, so as not to reduce the submitter's
voting power when voting on their own (and competing) actions. For a
```